app/pages/common/section/components/fields/combobox-options.njk

```diff
@@ -19,7 +19,7 @@
     } %}
   {% endif %}
 
-  <div id="{{ params.question.id }}" data-test-id="question-{{ params.question.id }}">
+  <div data-test-id="question-{{ params.question.id }}">
```

Comment:
I think we need the id to navigate to the anchor on error

Reply:
This is exactly how it works elsewhere

Reply:
yes we do and it needs to be in that format

Reply:
All of the input types are like this, the id is on the actual input (<select>, <input> etc.) not the input group (Labels, hint and input), I think this needs more thought than just adding it back in as the page then has 2 tags with the same id which breaks accessibility testing - it would be nice to have the anchor on the group rather than just the input but that needs to be fixed for all input types and would have to be something different to the input id. Does this make any sense at all?

Reply:
I agree it needs a second look as also the text areas just go to the inout box and don't even show the question

Reply:
could write a ticket for tech debt

Reply:
Yeah I'd count it as tech debt

Reply:
Sorry, just catching up. I don't mind creating techdebt tickets but I don't want this to bring back the bug that Katie already fixed.

Reply:
I think what I'm saying is adding the id into the wrapper doesn't fix the bug properly, it just creates a different bug. And removing it makes the bug consistent with all the other input components

Reply:
But I don't want to go down that rabbit hole as it isn't part of this story

Reply:
ok consistent bug is better. We should create a bug ticket for it for next month. Not really a tech-debt either